internal/ir/xds.go

```diff
@@ -12,15 +12,14 @@ import (
 	"net/netip"
 	"reflect"
 
+	egv1a1 "github.com/envoyproxy/gateway/api/v1alpha1"
+	egv1a1validation "github.com/envoyproxy/gateway/api/v1alpha1/validation"
 	"golang.org/x/exp/slices"
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/util/validation"
 	"sigs.k8s.io/yaml"
-
-	egv1a1 "github.com/envoyproxy/gateway/api/v1alpha1"
```

Comment:
internal packages should always be defined after public packages

Reply:
fixed